### DISCUSSION TITLE
Restoring metadata for colored coins and other smart wallets from backup

### DISCUSSION THREAD

**Discussion by gtmarchesini** [Chia-Network/chia-blockchain#9732] [Support]
Dear group, 
I'm sure someone faced this issue before. I was building two plots (queued) and was reaching 20% on my first one when I tried to reload the window. The system told me that plotting would stop, and I agreed. After that, I had to reopen the Chia wallet, enter my mnemonic phrase and it asked me to restore metadata for colored coins and other smart wallets from backup (basically a drag and drop box). 
What files should I restore? IT has a green button saying it's "SAFE TO SKIP", but when I click on it, it doesn't work. 
I looked up the files on the drive I'm storing the data, but every file I drag and drop, it says "unable to decrypt the backup file". 
Can anyone help me with this? I tried dropping file extensions and none of them seem to work. 
I would really appreciate some help with this so I can get back to building my plots!
Thank you all

**Comment by Merallak** [comment]
+1
I need help with the same issue. I have 2XCH on my wallet and it has been blocked by this "glitch?" since XCH was 254USD  —or so—.
